modules/how-to-plan-your-environment-according-to-application-requirements.adoc

```diff
@@ -82,90 +82,90 @@ links in the deployment's service specification file to overcome this:
 
 [source,yaml]
 ----
- ---
-    Kind: Template
-    apiVersion: v1
-    metadata:
-      name: deploymentConfigTemplate
-      creationTimestamp:
-      annotations:
-        description: This template will create a deploymentConfig with 1 replica, 4 env vars and a service.
-        tags: ''
-    objects:
-    - kind: DeploymentConfig
-      apiVersion: v1
+---
+Kind: Template
+apiVersion: v1
+metadata:
+  name: deploymentConfigTemplate
```

Comment:
The Template "deploymentConfigTemplate" is invalid: metadata.name: Invalid value: "deploymentConfigTemplate": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')